### ISSUE TITLE
Re-weight activities

### ISSUE THREAD

**Issue by revsean** [samhowes/foothills-data#35]
Just adding this as a reminder for myself once all the activity type, and duplicate questions are figured out. Some of the weights seem to be a bit off, like joining a group is a 6?

**Comment by samhowes** [comment]
@revsean: correct, joining a group is weighted at 6: https://github.com/samhowes/foothills-data/blob/master/Orbit/Sync/sync.yaml#L17

This appears to be the specified weight in the original spreadsheet: https://docs.google.com/spreadsheets/d/1QPut9GNWdpLSl4SwP9HYHp5kbWqdZBlGn8Rd6P4k4Fc/edit#gid=437762257

Changing for future uploads will be as simple as updating that yaml file linked above. Updating already uploaded data will be a very small effort.
